content/learn/tested.adoc

```diff
@@ -0,0 +1,75 @@
+== tl;dr
+
+The Tested tier is intended to provide consumers with additional collateral and reassurance that the pattern was known to be recently working on at least one recent version of OpenShift.  Inclusion in this tier requires some additional work for the pattern's owner - which might be a partner or sufficiently motivated SME.
+
+[id="nominating-a-community-pattern-to-become-Tested"]
+== Nominating a Sandbox Pattern for Promotion to Tested
+
+If there is a Sandbox pattern that you believe would be a good candidate for promotion to tested, you can make that case by emailing validatedpatterns@googlegroups.com
+
+Please be aware that each Tested pattern represents an ongoing maintenance, support, and testing effort.  Finite team capacity means that it is simply not possible for the team to take on this responsibility for all Validated Patterns.  
+
+For this reason we have designed the tiers and our processes to facilitate this to occur outside of the team by any sufficiently motivated party, including other parts of Red Hat, partners, and even customers.
+
+In limited cases, the Validated Patterns team may consider taking on that work, however please get in contact at least 4 weeks prior to the end of a given quarter in order for the necessary work to be considered as part of the following quarter's planning process
+
+
+[id="requirements"]
+== Requirements
+
+Tested Patterns have deliverable and requirements *in addition* to those
+specified for the link:/requirements/sandbox/[Sandbox tier]
+
+[id="must"]
+=== Must
+
+. Tested Patterns *MUST* continue to meet the following criteria to remain in the Tested tier
+. Tested Patterns *MUST* conform to the common technical link:/requirements/implementation/[implementation requirements]
+. Tested Patterns *MUST* be meaningful without specialized hardware, including flavors of architectures not explicitly supported.
++
+Qualification is a Validated Patterns TOC decision with input from the pattern owner.
+
+. Tested Patterns *MUST* have their implementation reviewed by the patterns team to ensure that it is sufficiently flexible to function across a variety of platforms, customer environments, and any relevant verticals.
+. Tested Patterns *MUST* include a standardized architecture drawing, created with (or at least conforming to) the PAC tooling
+. Tested Patterns *MUST* include a written guide for others to follow when demonstrating the pattern
+. Tested Patterns *MUST* include a test plan covering all features or attributes being highlighted by the demonstration guide.  Negative flow tests (such as resiliency or data retention in the presence of network outages) are also limited to scenarios covered by the demonstration guide.
++
+The test plan *MUST* define how to validate if the pattern has been successfully deployed and is functionally operational.
+Example: https://docs.google.com/document/d/12KQhdzjVIsxRURTnWAckiEMB3_96oWBjtlTXi1q73cg/view[Validating an Industrial Edge Deployment]
```

Comment:
ENOACCESS for non rh folks

Reply:
Yep, hence the subsequent "convert to adoc" comment

Reply:
Ack, missed that